src/data/markdown/docs/30 xk6-browser/01 xk6-browser/07 Locator/textcontent--options--.md

```diff
@@ -0,0 +1,35 @@
+---
+title: 'textContent([options])'
+excerpt: 'xk6-browser: locator.textContent method'
+---
+
+Returns the `element.textContent`.
```

Comment:
Probably just me, but how is this `element.textContent` different from `element.innerText`?

Reply:
There is a difference and I think I will leave it to the internet to help answer it: https://stackoverflow.com/questions/35213147/difference-between-textcontent-vs-innertext 🙂